### ISSUE TITLE
[REQUEST] please have popup alert if page will try to open new Tabwindow or external app

### ISSUE THREAD

**Issue by BaceCells** [tuyafeng/Via#645]
[REQUEST] please have popup alert if page will try to open new Tabwindow or external app
![IMG_20201101_000921](https://user-images.githubusercontent.com/73751864/97784068-d840aa80-1bd6-11eb-84c8-41afb50e98fb.jpg)

**Comment by tuyafeng** [comment]
Is this necessary?  Because this seems to interrupt people's browsing experience.

**Comment by Infamous-Jr** [comment]
> Is this necessary? Because this seems to interrupt people's browsing experience.

Yes, A pop-up & redirect blocker is must I know the browser has a option to block pop-up but it isn't working. 
For eg whenever I watch a video & click on the Play button the the website forced redirect to a new tab which is ads or some money betting sites & many more.
You can test it out yourself on the website 9anime-tv.com
![Screenshot_2020-11-06-23-05-35-813_com oh bro](https://user-images.githubusercontent.com/68345385/98674311-a372e700-237e-11eb-8a41-07846aab5d1c.jpg)

If possible please implement something like this as shown in the above image if the user doesn't allow the pop-up to open(because they are ads) the pop up blocker would automatically decline the website request to open the redirect within a time frame of 5-7 sec if no action has been taken.

And if you still think that it will effect the user experience then please give an option to enable/disable it from the settings. If the other user doesn't like this they can disable it.
